test/collections/componentSetBuilder.test.ts

```diff
@@ -623,6 +623,63 @@ describe('ComponentSetBuilder', () => {
         expect(compSet.has(apexClassWildcardMatch)).to.equal(true);
       });
 
+      it('should exclude folder type when excluding a type that is stored in folders', async () => {
```

Comment:
can we add a test that excludes `DasbhoardFolder` and makes sure `Dashboard` is still included?